src/easyreflectometry/project.py

```diff
@@ -44,24 +44,27 @@ def reset(self):
         self._materials = MaterialCollection(populate_if_none=False, unique_name='project_materials')
 
         self._info = self._default_info()
-        self._path = Path(os.path.expanduser('~'))
+        self._root_path = Path(os.path.expanduser('~'))
         self._calculator = None
         self._minimizer = None
         self._experiments = None
         self._colors = None
         self._report = None
 
         # Project flags
-        self._project_created = False
-        self._project_with_experiments = False
+        self._created = False
+        self._with_experiments = False
+
+    @property
+    def created(self) -> bool:
+        return self._created
 
     @property
     def path(self):
-        return self._path
+        return self._root_path / self._info['name']
```

Comment:
this will fail if the name contains `/` (or `\` on windows), e.g.

`self._info['name'] = 'my great project/invention'` 

will try to traverse to `my great project` directory which has not been created

Reply:
You have a good point.  However, I will leave the code as is for now